Packs/MicrosoftGraphMail/ReleaseNotes/1_5_15.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+##### Microsoft Graph Mail Single User
+- Fixed an issue where HTML body of emails was not being retrieved by the integration.
```

Comment:
I recommend modifying the release type since this change adds a new feature rather than fixing a bug.

Reply:
done in PR 29839.